packages/slint-lsp/package.yaml

```
@@ -19,6 +19,10 @@ source:
     - target: win_x64
       file: slint-lsp-windows.zip
       bin: slint-lsp/slint-lsp.exe
+    - target: [darwin_x86, darwin_arm64]
+      build:
+        run: cargo install slint-lsp
+      bin: slint-lsp
```

Since we were already building binaries for macOS, I took the liberty of adding them the 1.3.2 release at https://github.com/slint-ui/slint/releases/tag/v1.3.2 . Those `-macos.tar.gz` archives were produced in the same build as the linux/windows once. They are not code signed though, but from what I can tell other macOS binaries in the mason registry don't do that either (for example tree-sitter).

Could you try the suggestion below?

```suggestion
    - target: [darwin_x86, darwin_arm64]
      file: slint-lsp-macos.tar.gz
      bin: slint-lsp/slint-lsp
 ```